Description: Fix path location for templates and other files
Author: Laurent Bigonville <bigon@debian.org>
Forwarded: not-needed

--- a/lib/vagrant.rb
+++ b/lib/vagrant.rb
@@ -114,7 +114,7 @@ module Vagrant
   # The source root is the path to the root directory of
   # the Vagrant gem.
   def self.source_root
-    @source_root ||= Pathname.new(File.expand_path('../../', __FILE__))
+    @source_root ||= Pathname.new('/usr/share/vagrant/')
   end
 
   # Configure a Vagrant environment. The version specifies the version
