Description: mount NFS with misc option
Bug: https://github.com/mitchellh/vagrant/pull/1029
Author: Marc Fournier <marc.fournier@camptocamp.com>

--- a/plugins/guests/linux/cap/mount_nfs.rb
+++ b/plugins/guests/linux/cap/mount_nfs.rb
@@ -18,7 +18,7 @@ module VagrantPlugins
             # Mount
             hostpath = opts[:hostpath].dup
             hostpath.gsub!("'", "'\\\\''")
-            mount_command = "mount -o vers=#{opts[:nfs_version]} #{ip}:'#{hostpath}' #{expanded_guest_path}"
+            mount_command = "mount -o udp,noatime,nodiratime,rsize=32768,wsize=32768,vers=#{opts[:nfs_version]} #{ip}:'#{hostpath}' #{expanded_guest_path}"
 
             retryable(:on => Vagrant::Errors::LinuxNFSMountFailed, :tries => 5, :sleep => 2) do
               machine.communicate.sudo(mount_command,
