Description: puppet provisioner NFS support
Bug: https://github.com/mitchellh/vagrant/issues/1862
Author: Marc Fournier <marc.fournier@camptocamp.com>

--- a/plugins/provisioners/puppet/config/puppet.rb
+++ b/plugins/provisioners/puppet/config/puppet.rb
@@ -10,6 +10,7 @@ module VagrantPlugins
         attr_accessor :options
         attr_accessor :temp_dir
         attr_accessor :working_directory
+        attr_accessor :nfs
 
         def initialize
           super
@@ -22,6 +23,7 @@ module VagrantPlugins
           @facter            = {}
           @temp_dir          = UNSET_VALUE
           @working_directory = UNSET_VALUE
+          @nfs               = UNSET_VALUE
         end
 
         def finalize!
@@ -33,6 +35,7 @@ module VagrantPlugins
           @module_path    = nil if @module_path == UNSET_VALUE
           @temp_dir       = "/tmp/vagrant-puppet" if @temp_dir == UNSET_VALUE
           @working_directory = nil if @working_directory == UNSET_VALUE
+          @nfs            = false if @nfs == UNSET_VALUE
         end
 
         # Returns the manifests path expanded relative to the root path of the
--- a/plugins/provisioners/puppet/provisioner/puppet.rb
+++ b/plugins/provisioners/puppet/provisioner/puppet.rb
@@ -29,14 +29,14 @@ module VagrantPlugins
 
           # Share the manifests directory with the guest
           root_config.vm.synced_folder(
-            @expanded_manifests_path, manifests_guest_path)
+            @expanded_manifests_path, manifests_guest_path, :nfs => @config.nfs)
 
           # Share the module paths
           count = 0
           @module_paths.each do |from, to|
             # Sorry for the cryptic key here, but VirtualBox has a strange limit on
             # maximum size for it and its something small (around 10)
-            root_config.vm.synced_folder(from, to)
+            root_config.vm.synced_folder(from, to, :nfs => @config.nfs)
             count += 1
           end
         end
